Jenkinsfile

```diff
@@ -626,6 +627,96 @@ pipeline {
                         }
                     }
                 }
+                stage('arm64') {
+                    when {
+                        beforeAgent true
+                        expression { params.arm64 }
+                    }
+                    agent { label 'aarch64' }
```

Comment:
you might want to make this `arm64&&linux`, just to specify that these workloads are specific for linux

Reply:
Ah; wasn't sure if we had a `arm64` label

good point on the `linux` part (although I guess we don't have Windows ARM running)